scripts/build

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+set -e
+
 npm_bin=$(npm bin)
 
+echo "Cleaning build"
+rm -rf build/*
```

Comment:
I'd rather

```bash
rm -rf build
mkdir build
```

Reasons being:

1. any dotfiles won't be removed with `build/*` (which could include caches)
2. I don't like `*` or `/` in `rm -rf` commands that other people are expected to run

---

Also this command doesn't set it's cwd, so if you were to call it from somewhere else (e.g. in a crontab file configured to run a build every day at 4am) then it would be deleting a different folder. I'd rather see this paired with something like this to change into the project directory first:

```bash
cd "$(dirname "$(dirname "$(realpath "$0")")")"
```